src/main/java/com/example/onlinebookstore/dto/order/OrderUpdateDto.java

```diff
@@ -0,0 +1,13 @@
+package com.example.onlinebookstore.dto.order;
+
+import com.example.onlinebookstore.model.Order;
+import jakarta.validation.constraints.NotBlank;
+
+public record OrderUpdateDto(
+
+        @NotBlank
+        String shippingAddress,
```

Comment:
you don't need shippingAddress in this request. The requirements are said about update status